src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -493,3 +496,20 @@ void CalculatorApp::GraphingCalculator::ActiveTracing_KeyUp(Windows::UI::Core::C
         args->Handled = true;
     }
 }
+
+void GraphingCalculator::GraphSettingsButton_Click(Object ^ sender, RoutedEventArgs ^ e)
+{
+    if (m_flyoutGraphSettings == nullptr)
```

Comment:
Why doesn't this initialization happen somewhere more central? Would protect against other methods of editing the graph settings in the future such as from a hot key, etc.